lib/handler.js

```diff
@@ -65,10 +65,14 @@ module.exports = function(options) {
             actionDescriptionObj.versionType = 'external';
           }
 
-          const doc = options.pickFields
+          let doc = options.pickFields
             ? _.pick(parsedRecord.NewImage, options.pickFields)
             : parsedRecord.NewImage;
 
+          if(options.processRecordHook){
+            doc = options.processRecordHook(doc);
```

Comment:
It may be not a bad idea to add check like
```js
if (!doc) {
  throw new Error('processRecordHook must return object');
}
```
Just as a safeguard. Thoughts?